TravelExpertsGUI/frmMain.cs

```diff
@@ -1,26 +1,57 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.ComponentModel;
 using System.Data;
-using System.Drawing;
 using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 using System.Windows.Forms;
+using TravelExpertsData;
+using TravelExpertsDataAPI;
 
 namespace TravelExpertsGUI
 {
     public partial class frmMain : Form
     {
+        // productKeyMap maps the ListBox indices to the ProductId from the database
+        Dictionary<int, int> productKeyMap = new Dictionary<int, int>();
+
+        // A list of all products in the database
+        List<Product> products;
+
         // WOO new repo on branch!
         public frmMain()
         {
             InitializeComponent();
         }
 
-        private void Form1_Load(object sender, EventArgs e)
+        private void frmMain_Load(object sender, EventArgs e)
         {
-            
+            // Product panel code
+            int counter = 0;
+            products = ProductDB.GetProducts()
+                .OrderBy(p => p.ProdName)
+                .ToList();
+            products.ForEach( 
+                p => {
+                    lstProductTabProducts.Items.Add(p.ProdName);
+                    productKeyMap.Add(counter, p.ProductId);
+                    counter++;
+                }
+            );
+        }
+
+        private void lstProducts_SelectedIndexChanged(object sender, EventArgs e)
```

Comment:
Name `lstProducts` is to ambiguous. Please change to `lstProductsTabProducts` for better clarity